AtbIntegrationTests/AtbIntegrationTests.swift

```diff
@@ -202,12 +205,21 @@ class AtbIntegrationTests: XCTestCase {
         }
         
         server["/atb.js"] = {
-            self.atbRequests.append($0)
+            var atb = ""
```

Comment:
This logic makes me nervous.  Rather than putting in to an appropriate bucket by type why not assert requests are the type that are expected in the order that is expected?

Reply:
@brindy that's a good point, testing the overall request count and order would be preferable. I've refactored the code to do that.